.eslintrc

```diff
@@ -59,6 +59,20 @@
         },
         "allowChildren": false
       }
-    ]
+    ],
+    "react/sort-comp": ["error", {
```

Comment:
any particular reason why this is part of this PR?
if it was a group decision, then ignore this comment, if it wasn't I think it should be discussed and add it to the decision folder

Reply:
No I just didn't consider the option of putting the flow attributes below render since that just seemed silly. Happy to discuss it further though